examples/flutter_gallery/lib/demo/material/search_demo.dart

```diff
@@ -133,6 +133,12 @@ class _SearchDemoState extends State<SearchDemo> {
   }
 }
 
+/// [SearchDelegate.buildResults] might be called multiple times. Please cache
+/// the search result if you don't want to call search backend multiple times
+/// on the same query.
+/// See:
```

Comment:
We usually don't include issue references like this in doc comments.

Reply:
Removed.